plugins/catalog/src/components/CatalogTable/columns.tsx

```diff
@@ -25,6 +25,31 @@ import { OverflowTooltip, TableColumn } from '@backstage/core-components';
 import { Entity } from '@backstage/catalog-model';
 import { JsonArray } from '@backstage/types';
 
+/**
+ * Renders label if exists in entities or default
+ * @param labels - Key value pairs of labels specified in entity metadata
+ * @param key - specified label key to be extracted from all labels
+ * @param defaultValue - shows default in case label key does not exist
+ * @returns Chip style component when label value exists undefined otherwise.
+ */
+const renderLabel = (
```

Comment:
As this function is used in one place let's instead inline the code into  `createLabelColumn` :)

Reply:
Yep I thought it as well, done in latest commit :)